packages/strapi-design-system/src/Button/Button.js

```diff
@@ -27,19 +27,33 @@ const BoxFullHeight = styled(Box)`
   height: 100%;
 `;
 
-// TODO: Check the L size button with Maeva
 export const ButtonWrapper = styled(BaseButton)`
   align-items: center;
-  padding: ${({ theme, size }) => `${size === 'S' ? theme.spaces[2] : '10px'} ${theme.spaces[4]}`};
-  background: ${({ theme }) => theme.colors.buttonPrimary600};
+  background-color: ${({ theme }) => theme.colors.buttonPrimary600};
   border: 1px solid ${({ theme }) => theme.colors.buttonPrimary600};
+  height: ${({ size }) => {
+    let height = 32;
+
+    if (size === 'M') {
+      height = 36;
+    } else if (size === 'L') {
+      height = 40;
+    }
+
+    return `${height / 16}rem`;
+  }};
```

Comment:
#### Suggestion

IMO, will be good to put the button sizes at the file `src/themes/sizes.js`, like other components.
WDYT?

```suggestion
  height: ${({ size, theme }) => theme.sizes.button[size] || themes.sizes.button.S};
```

😂 _That snippet could not work, I'm not sure if will work properly_

Reply:
Done in 875e6bc65b10f2beed1cac6deb09f1739922e415. Thanks - that's a very good idea!